tasks/orchestrator.yml

```diff
@@ -38,3 +38,11 @@
   when: yaml_imagestream.changed
   tags:
   - oc
+
+- name: update imagestream tag
+  command: >
+    oc import-image {{ osbs_buildroot_imagestream }}:{{ osbs_buildroot_new_tag }} --from {{ osbs_buildroot_repository }}{{ " --insecure" if osbs_insecure_repository else ''}}
```

Comment:
Let's break this line up into multiple lines.

Reply:
sure